.github/workflows/entrypoint_pull_request.yml

```diff
@@ -44,9 +44,57 @@ jobs:
           echo "IMAGE_BASE_NAME=PR${PR_NUMBER}" >> $GITHUB_OUTPUT
           echo "DOCKERFILE=${DOCKERFILE}" >> $GITHUB_OUTPUT
 
+  setup-code-check:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install Rust and Cargo
+      uses: actions-rs/toolchain@v1
```

Comment:
This action is deprecated and hasn't been maintained since late 2020. This causes some warnings in the actions logs like
- `The following actions uses node12 which is deprecated and will be forced to run on node16`
- `The `set-output` command is deprecated and will be disabled soon`

We need to find an alternative, more maintained action, or do it ourselves.